test/components/paginatable_component_test.rb

```diff
@@ -0,0 +1,10 @@
+require "test_helper"
```

Comment:
I'd love to add tests, if possible. I'm happy to go down the route of a real integration test with a browser and JavaScript and all that for this one if need be.

Reply:
Do you have any thoughts on JavaScript-powered tests for this?

Reply:
I don't currently have thoughts on system tests. It definitely should be done, but I figure it was untested code before and at least it will be harder to break with the logic in a component now. I'm sorry I don't have a more definitive answer now :/

Reply:
Fair enough. And good point – happy to merge these changes in and create an issue for JS tests in the future.